src/app/service/scoap-squad/BotConversationMessageFlow.ts

```diff
@@ -0,0 +1,226 @@
+import { publishDraftScoapEmbed } from './CreateNewScoapPoll';
+import constants from '../constants/constants';
+import { ScoapEmbed } from './ScoapClasses';
+import { Message } from 'discord.js';
+import ScoapUtils from '../../utils/ScoapUtils';
+import { scoapEmbedState, botConvoState, voteRecordState } from './ScoapDatabase';
+
+
+export default async (message: Message, botConvo: any): Promise<any> => {
+	switch (true) {
+	case (botConvo.getCurrentMessageFlowIndex() === '2'):
+		switch (true) {
+		case (ScoapUtils.validateTitle(message.content)):
+			if (!botConvo.getEdit()) {
+				setUserResponseRecord(message.content, botConvo, 'TITLE');
+			}
+			incrementMessageFlowIndex(botConvo, message, ['CORRECT', 1]);
+			return;
+		default:
+			incrementMessageFlowIndex(botConvo, message, ['INCORRECT', '\n- 250 characters maximum\n ' +
+																		'- alphanumeric\n ' +
+																		'- special characters: .!@#$%&,?\n']);
+			return;
+		}
+		break;
```

Comment:
It's not possible to reach this `break` statement because of lines 18 and 23. We can probably leave it in anyways but please check the missing `await`s on line 17,20,32,35,47,50,61,64,79,82,99,102,112, 180,183,186,

Reply:
Are we using an eslint plugin? I've seen eslint added to the project. These problems can be caught by eslint (or sonar) and we can break the build if something like this happens.

Reply:
Strange, I am pretty sure it was eslint that told me to keep the break statement in there. Now removed it and no complaints from eslint :/

Reply:
I only added await in lines 177, 180, 183 (formerly 180,183,186,) because of the above mentioned behavior.